tests/phpunit/tests/plugins/test-wp-importer.php

```diff
@@ -13,7 +13,6 @@ public function set_up() {
 		require_once POLYLANG_DIR . '/include/api.php';
 
 		self::$model->options['hide_default'] = 0;
-		update_option( 'polylang', self::$model->options ); // make sure we have options in DB ( needed by PLL_WP_Import )
```

Comment:
Does the constraint in the comment is still met?
Because `shutdown` is run waaaay before any tests are run I think. SO not sure options are already saved.

Reply:
As far that I can tell, the comment is a bit misleading with these new options classes. The options do not have to be in the DB, they have to be available in `PLL_WP_Import`, which uses `PLL()->options`.

Then, we have this in `set_up()`:
```php
self::$model->options['hide_default'] = 0;
// ...
$links_model = self::$model->get_links_model();
$pll_admin = new PLL_Admin( $links_model );
$GLOBALS['polylang'] = &$pll_admin;
```

`$GLOBALS['polylang']` is built by using `self::$model->options`.

Reply:
Okay 👍